src/pages/Environments.js

```diff
@@ -425,9 +425,13 @@ const Environments = () => {
               const multipleDisksOfType = _.remove(disk => getDiskAppType(disk) !== appType || disk.status === 'Deleting',
                 disksByProject[googleProject]).length > 1
               const forAppText = !!appType ? ` for ${_.capitalize(appType)}` : ''
+              // Galaxy can have multiple disks per project as long as they are in different workspaces
+              const galaxyDisksOnly = _.filter(disk => getDiskAppType(disk) === tools.galaxy.appType && disk.status !== 'DELETING', filteredDisks)
```

Comment:
Looks like we had already gotten the app type. Can we use that instead of getting it again?
```suggestion
              const galaxyDisksOnly = _.filter(disk => appType === tools.galaxy.appType && disk.status !== 'DELETING', filteredDisks)
```

Reply:
I think this check should not be Galaxy-specific.